### PR TITLE
Feature/add requests tests

### PR DESCRIPTION
**Pull request by jvargas98**

### CODE REVIEW

File: spec/requests/alarms_spec.rb [hunk 1]
```diff
@@ -0,0 +1,168 @@
+require 'rails_helper'
+
```
Comment (OmauryMarmolejo) [inline spec/requests/alarms_spec.rb:2]: Cool!

File: spec/requests/stopwatches_spec.rb [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+  describe 'POST #create' do
+    let(:stopwatch_valid_params) do
+      {
+        time: '2021/17/03 13:00:00',
+        name: 'Stopwatch test',
+        laps: ['00:30:00', '00:25:00'],
+        user_id: user.id
+      }
+    end
```
Comment (OmauryMarmolejo) [inline spec/requests/stopwatches_spec.rb]: Avoid hardcoded values; you can generate these values randomly, defining a pattern for them.

**Review by OmauryMarmolejo — COMMENTED** [review]
Great job Jorge!

**Review by ese-varo — APPROVED** [review]

File: spec/requests/users_spec.rb [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+        it 'update with invalid attributes' do
+          patch user_path(user), params: { id: user.id, user: { time_zone: nil } }
+          expect(response).to redirect_to(root_path)
+        end
+      end
+    end
+
+    context 'when user is not signed in' do
+    end
```
Comment (ese-varo) [inline spec/requests/users_spec.rb:100]: Are you adding specs for this context, if not maybe you could just remove it or skip it:
```suggestion
    context 'when user is not signed in', :skip do
    end
```
or if you plan to add them you could mark it as pending to remind you that those aren't implemented yet: 
```suggestion
    context 'when user is not signed in', :pending do
    end
```

**Review by ese-varo — APPROVED** [review]
Great job Vargas!! :rocket: I just left some observations, but actually it looks nice! :muscle:

**Review by LoginALee — COMMENTED** [review]
Good work Jorge :100:  !

File: spec/requests/errors_spec.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+      it 'returns a success response' do
+        get :not_found
+        expect(response.body).to include('Not Found')
+      end
+    end
+    context '#unprocessable_entity' do
+    end
+
+    context '#server_error' do
+    end
```
Comment (LoginALee) [inline spec/requests/errors_spec.rb]: You can delete that contexts if they are empty

**Pull request merged**